jwql/website/apps/jwql/templates/api_landing.html

```diff
@@ -49,8 +49,8 @@ <h5>Obtaining Filenames Associated with a Rootname</h5>
                     from urllib import request
                     import json
 
-                    # Build the URL https://dljwql.stsci.edu/api/roontname/service/
-                    url = 'https://dljwql.stsci.edu/api/jw93025001001_02102_00001_nrca2/filenames/'
+                    # Build the URL https://jwql.stsci.edu/api/roontname/service/
+                    url = 'https://jwql.stsci.edu/api/jw93025001001_02102_00001_nrca2/filenames/'
```

Comment:
Are we moving to the production server now?!

Reply:
Yeah, I would like the users to use the production server for this service, so best to use it in the examples.